torchvision/prototype/transforms/_geometry.py

```diff
@@ -194,31 +194,22 @@ def forward(self, *inputs: Any) -> Any:
 
 
 class BatchMultiCrop(Transform):
+    def _batch(self, crops: Any) -> Any:
```

Comment:
Nit: Why not name it `_transform()` as with every other class? It just happens that this Transform is doing batching.

Reply:
Just my misconception that we can't use the regular idiom for `MultiCropResult`. But `pytree` treats it differently than regular `list`'s or `tuple`'s and thus we can use it after all. Thanks for the heads-up.